### ISSUE TITLE
Resolution changes due to bandwith problems -> malformed video stream

### ISSUE THREAD

**Issue by jeryini** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-js#94]
I'm having issue with video stream when there is apparently resolution change from the master streamer due to bandwidth problems (degraded mobile network connection strength). What happens is that video becomes "malformed", which is supposed due to resolution change. See example:
![webrtc-corrupt](https://user-images.githubusercontent.com/1238022/92301825-22a30380-ef67-11ea-8111-a9c1311e93dc.jpg)

Apparently this issue is due to video player still displaying the same resolution even though master streamer has changed the resolution. Is this something that can be configured in this library or is it completely in the hands of a video player?

**Comment by lherman-cs** [comment]
@jeryini did you use the embedded SDK or JS SDK for the viewer? If JS SDK, this can't be configured in the SDK, you need to go to the browser APIs or possibly a third party video player.

**Comment by jeryini** [comment]
Hello @lherman-cs ! I've used JS SDK for the viewer. I've also posted [this question](https://stackoverflow.com/questions/63767434/html5-video-and-mediastreamtrack-stream-via-webrtc-resolution-change-issue/63772574#63772574) on SO and provided some more details. I'll also write them here.

Debugging on the `MediaStreamTrack` in console shows how resolution has changed:
![Image of Console log](https://i.stack.imgur.com/xdDpg.png)

But here comes the interesting part. It seems that native HTML5 video player has problems only with certain resolutions. In this case it was working fine with 540x720px and 1080x1440px, but not with 810x1080px. I have browsed the web for native HTML5 video player specification but there is nothing on supported resolutions.

It seems that this might be a bug directly in browser.
